server/dash/dashboards/neighborhood_recent.py

```diff
@@ -4,13 +4,15 @@
 import dash_core_components as dcc
 import dash_html_components as html
 import dash_table
+from dash.dependencies import Input, Output
 import pandas as pd
 import plotly.express as px
+from flask import request
```

Comment:
This doesn't look like it's being used, so we can remove it.

Reply:
Thanks for the catch - resolved!